rskj-core/src/main/java/co/rsk/peg/BridgeSupport.java

```diff
@@ -1019,13 +1019,17 @@ private void logUpdateCollections(Transaction rskTx) {
 
     private boolean svpIsOngoing() {
         return federationSupport.getProposedFederation()
-            .map(Federation::getCreationBlockNumber)
-            .map(proposedFederationCreationBlockNumber ->
-                proposedFederationCreationBlockNumber + bridgeConstants.getFederationConstants().getValidationPeriodDurationInBlocks())
-            .filter(validationPeriodEndBlock -> rskExecutionBlock.getNumber() <= validationPeriodEndBlock)
+            .filter(this::validationPeriodIsOngoing)
             .isPresent();
     }
 
+    private boolean validationPeriodIsOngoing(Federation proposedFederation) {
+        long validationPeriodEndBlock = proposedFederation.getCreationBlockNumber() +
+            bridgeConstants.getFederationConstants().getValidationPeriodDurationInBlocks();
+
+        return rskExecutionBlock.getNumber() < validationPeriodEndBlock;
```

Comment:
also replaced the `<=` for a `<` to follow the same logic as in migration period (i.e. not considering the ending block as part of the period)